src/Microsoft.Azure.WebJobs.Host/Hosting/IWebJobsStartup.cs

```diff
@@ -1,6 +1,8 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
+using System;
```

Comment:
Chose to not mark these startup interfaces as Obsolete. @fabiocav any objections?

Reply:
No objections.